src/component/footer/footer.js

```diff
@@ -1,25 +1,15 @@
 import React from 'react';
+
 import CacheButton from './cache.js';
 import Throttle from './throttle.js';
 import RequestCount from './request-count.js';
 import ProxyStatus from './proxy-status.js';
 
-const {func, object} = React.PropTypes;
-
-const Footer = (props) => {
-  return <div className="footer">
-    <CacheButton {...props} />
-    <Throttle {...props} />
-    <RequestCount {...props} />
-    <ProxyStatus {...props} />
-  </div>;
-};
-
-Footer.propTypes = {
-  toggleCaching: func.isRequired,
-  isCachingEnabled: func.isRequired,
-  clearRequests: func.isRequired,
-  requestData: object.isRequired
-};
+const Footer = () => <div className="footer">
```

Comment:
To explain this, redux's store emits updates on changes and `connect`ed components subscribe to those updates, updating as necessary.

`connect` creates a higher-order component (HOC) which has a pretty solid `shouldComponentUpdate` to prevent any extra changes, so it's a pretty decent fit in this case, I think.